src/components/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,86 @@
+'use client'
+import React from 'react'
+import styles from './ProfileDetails.module.scss'
+import { useSession } from 'next-auth/react'
+
+const ProfileDetails = () => {
```

Comment:
**thought:** Looks like we are missing component parametrization by props, but it can probably be not necessary because of using server components later when we have API, and data ready.